assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -182,69 +222,97 @@ function MediaPane(props) {
     )
     .map((attachment) => getResourceFromAttachment(attachment));
 
+  const refContainer = useRef();
+  const refContainerFooter = useRef();
+
+  useIntersectionEffect(
+    refContainerFooter,
+    (entry) => {
+      if (!isMediaLoaded || isMediaLoading) return;
```

Comment:
I think the style requires `{ return }` with new lines and all right now. Though if it doesn't - feel free to leave as is.